app/views/core/VueComponentView.js

```diff
@@ -10,14 +9,16 @@ const silentStore = { commit: _.noop, dispatch: _.noop }
 module.exports = class VueComponentView extends RootView {
```

Comment:
VueComponnetView and SubVueCompnentView are close to identical in terms of functionality with the exception that they inherit from different base classes and thus render differently.  I don't see a way to avoid having two of them but couldn't think of an easy way to share functionality outside of a mixin and didn't want to spend the time doing that for a prototype.

If we move forward with this we should discuss ways to consolidate.

Reply:
I agree that both these are identical and could possibly be consolidated. I would like to understand the rationale behind inheriting from CocoView instead of RootView for SubVueCompnentView